pytest.ini

```diff
@@ -0,0 +1,13 @@
+# pytest.ini
+[pytest]
+minversion = 6.0
+addopts = -ra -q
+norecursedirs = .svn _build ndarray* # ignore tests in ndarray due to missing stdVector module not found
+testpaths =
+    tests
+python_files = *_TestCase.py histogramTest_*.py #manually run python alltests.py will also return 145 tests
+filterwarnings =
```

Comment:
Future warnings will not be captured and showed, if/when we make changes in the code.
Let's think about this and find an alternative, if we really need to filter the warnings.

Reply:
This has been fixed. The cause is in Loader.py line 133 if "storage" in list(dataGroup()):, list(dataGroup()) is of type list(numpy numeric strings), it's comparing a raw python string with numpy string. Solution is convert list(dataGroup()) to python strings by adding str() wrapper